README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion: A method that calls itself.
+- Recursive Case: The smaller problem that calls itself till the base case is hit.
+- Base Case: The easiest case.
+- Activation Chain/Stack: The process when the method calls itself. (All the bubbles. The last bubble, before going up the chain again is the base case.)
+- Activation Record/Call: Each call (One of the bubbles)
+- Infinite Recursion/Stack Overflow/Stack too deep: When the base case is never reached.
+- Tail Recursion: 
```

Comment:
We didn't actually cover tail recursion in class. We may in the future, but it's not necessary to know this moment.